cirq-core/cirq/ops/raw_types.py

```diff
@@ -421,6 +423,35 @@ def _json_dict_(self) -> Dict[str, Any]:
         return protocols.obj_to_dict_helper(self, attribute_names=[])
 
 
+class Condition:
```

Comment:
Probably want this in `cirq-core/cirq/value/condition` Possibly rename to `ClassicalControlCondition`.

Reply:
Also, seems ripe for dataclasses